src/test/config.sol

```diff
@@ -157,18 +157,18 @@ contract Config {
             flip_ttl:     0,               // In seconds
             flip_tau:     0,               // In seconds
             flipper_mom:  0,               // 1 if circuit breaker enabled
-            dog_hole:     65 * MILLION,
-            clip_buf:     12000,
-            clip_tail:    140 minutes,
-            clip_cusp:    4000,
-            clip_chip:    10,
-            clip_tip:     300,
-            clipper_mom:  1,
-            cm_tolerance: 5000,
-            calc_tau:     0,
-            calc_step:    90,
-            calc_cut:     9900,
-            lerp:         false
+            dog_hole:     40 * MILLION,    // In whole Dai units
```

Comment:
Nice additions

Reply:
For disclosure these were Julien's additions :)